api/registry.go

```diff
@@ -90,6 +90,9 @@ func RegisterBatchAPI(root gin.IRouter) {
 	root.POST("/v4/collectibles/categories", func(c *gin.Context) {
 		endpoint.GetCollectionCategoriesFromList(c, platform.CollectionAPIs)
 	})
+	root.POST("/v3/tokens", func(c *gin.Context) {
```

Comment:
```suggestion
	root.POST("/v1/tokens", func(c *gin.Context) {
```

did we use v1 and v2 alredy?

Reply:
I think we can use `v2` here too